spring-cloud-skipper/src/main/java/org/springframework/cloud/skipper/domain/PackageMetadata.java

```diff
@@ -35,12 +36,16 @@
 	private String apiVersion;
 
 	/**
-	 * The repository ID this Package Index file belongs to. todo: Make this as Required once
-	 * we have repository backed package for upload etc.,
+	 * Indicates the origin of the repository (free form text).
 	 */
 	private String origin;
```

Comment:
origin is the repository Id and hence we may not need `repositoryId`

Reply:
I might be missing something here. What is the intention behind supporting both the `repositoryId` as well as the `origin` while we only had the `origin` pointing to the repository id before

Reply:
looking y'day discussion on slack, per mark's comments
```
If we add a new property to packageMetadata such as 'repositoryId', then that woudl be set here instead.
and in the other direction `Repository packageRepository = repositoryRepository.findOne(packageMetadata.getOrigin());`
would change to `Repository packageRepository = repositoryRepository.findOne(packageMetadata.getRepositoryId());`
```
looks like he meant replace, not just add new one?